src/main/java/Duke.java

```diff
@@ -1,10 +1,122 @@
+import java.util.*;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+  public static void main(String[] args) {
+    //        String logo = " ____        _        \n"
+    //                + "|  _ \\ _   _| | _____ \n"
+    //                + "| | | | | | | |/ / _ \\\n"
+    //                + "| |_| | |_| |   <  __/\n"
+    //                + "|____/ \\__,_|_|\\_\\___|\n";
+    //        System.out.println("Hello from\n" + logo);
+
+    String divider = "____________________________________________________________\n";
+    System.out.println(divider + "Hello! I'm Duke");
+    System.out.println("What can I do for you?\n" + divider);
+
+    ArrayList<Task> tasks = new ArrayList<>();
+    Scanner sc = new Scanner(System.in);
+
+    while(sc.hasNext()) {
+      String instr = sc.nextLine();
+      switch(instr) {                                 //Single-word instructions
+        case "list":  //List down
```

Comment:
Should indentation for `case` clauses be removed?
```suggestion
    case "list":
```